server/ServerApp.cpp

```diff
@@ -318,7 +318,6 @@ int ServerApp::GetNewDesignID()
 
 void ServerApp::Run() {
     DebugLogger() << "FreeOrion server waiting for network events";
-    std::cout << "FreeOrion server waiting for network events" << std::endl;
```

Comment:
Why?

With this line, there is some indication of what's happening if the server process is run from command line.